### PR TITLE
prepare 10.11.0-rc.1 docker image

### PR DESCRIPTION
**Pull request by jnweiger**
release 10.11.0-rc.1 with these apps update since 10.0.0.10
* customgroups: v0.6.2 - v0.7.0
* encryption v1.5.1 - v1.5.3
* example-files v1.2.0 - v1.3.0
* graphapi v0.2.0 - v0.3.0
* files_antivirus v1.0.0 - v1.1.0
* files_lifecycle v1.3.2 - v1.3.3
* guests v0.9.3 - v0.11.0
* market v0.6.3 - v0.7.0
* web v5.4.0 - v5.6.1

We don't build docker for RCs normally, @pmaier1 needs one, this time.

### CODE REVIEW

**Comment by xoxys** [comment]
Please fix the CI.

File: README.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ ownCloud is an open-source file sync, share and content collaboration software t
 ## Docker Tags and respective Dockerfile links
 
 - [`latest`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:latest`
+- [`10.11.0-rc.1`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.11.0-rc.1`
```
Comment (xoxys) [inline README.md:33]: I would like to not put RC's in the Readme and keep the supported stable tags only.

**Review by xoxys — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jnweiger) [reply README.md]: ok.

**Review by xoxys — CHANGES_REQUESTED** [review]
Please read my other comment as well... CI is still not working due to a syntax issue

**Comment by jnweiger** [comment]
CI fixed. Readme restored.

**Review by xoxys — APPROVED** [review]

**Comment by jnweiger** [comment]
Thanks, Robert!

**Comment by xoxys** [comment]
Thank you for preparing it :)

**Pull request merged**